migration/ament_tools.rst

```diff
@@ -91,6 +91,9 @@ ament test_results
 Behavioral changes
 ------------------
 
+The colcon ``test`` verb performs only the action of running tests.
+It does not configure, build or install targets.
```

Comment:
Since `colcon` doesn't really have the terms `configure`, `install`, or `targets` in the CLI how about changing this to:

> It does not build any packages.

Reply:
as colcon relies on build and install spaces and provide arguments such as "symlink-install" in the core, I figured we could uses those terms.
But I'm not attached to it and you're right that the notion of configuring and targets is pretty CMake / Make specific.